app/pb_kits/playbook/pb_time_range_inline/_time_range_inline.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= content_tag(:div,
+    id: object.id,
+    data: object.data,
+    class: object.classname) do %>
+      <div class="align-<%= object.alignment %>">
+        <% text_kit = "caption" %>
+        <% text_timezone_color = nil %>
+        <% if object.size == "sm" %>
+          <% text_kit = "body" %>
+          <% text_timezone_color = "light" %>
+        <% end %>
+        <% html_icon = "" %>
+        <% if object.icon == true %>
+          <% html_icon = pb_rails("icon", props: { icon: "clock", dark: object.dark, classname:"space-right", fixed_width: true, size: object.size }) %>
+        <% end %>
```

Comment:
We should move any variable assignments to methods with the same name. You won't have to pass any arguments as they will already be available via `object` in `app/pb_kits/playbook/pb_time_range_inline/time_range_inline.rb`

Reply:
@thestephenmarshall I don't know if the current version of the code satisfies this, but I couldn't find a way to remove the `if` logic from the `erb` file. Could you help me to achieve that? Thanks.